src/status_im/browser/core.cljs

```diff
@@ -82,16 +82,20 @@
   (let [history-host (http/url-host (try (nth history history-index) (catch js/Error _)))]
     (assoc browser :unsafe? (dependencies/phishing-detect history-host))))
 
+(def ipfs-proto-code "e3")
+
 (defn resolve-ens-content-callback [hex]
   (let [hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 2))))]
     (if (and hash (not= hash resolver/default-hash))
-      (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success "01" hash])
+      (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success ipfs-proto-code hash])
       (re-frame/dispatch [:browser.callback/resolve-ens-contenthash]))))
 
 (defn resolve-ens-contenthash-callback [hex]
   (let [proto-code (subs hex 2 4)
-        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 8))))]
-    (if (and (#{"00" "01"} proto-code) hash (not= hash resolver/default-hash))
+        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 12))))]
+    ;; We only support IPFS
+    ;; TODO Once more implementations / providers are published this will have to be improved
+    (if (and ((#{"00" ipfs-proto-code} proto-code) hash (not= hash resolver/default-hash)))
```

Comment:
are you sure swarm code is 00? looks like it should be e4. also it need to be changed in `resolve-ens-multihash-success`

Reply:
Fixed!